### PR TITLE
[PURCHASE-2249] First inquiry visual qa

### PR DESCRIPTION
**Pull request by xtina-starr**
The type of this PR is: **Fix**

Screen Shot 2020-12-02 at 2 32 32 PM

This PR resolves [PURCHASE-2249]

This vertically centers the chevron to the left of the 'Add your location' copy.

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [ ] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [ ] I have documented any follow-up work that this PR will require, or it does not require any.
- [ ] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [ ] I have added a [CHANGELOG.yml](/CHANGELOG.yml) entry or my changes do not require one.

[PURCHASE-2249]: https://artsyproduct.atlassian.net/browse/PURCHASE-2249

### CODE REVIEW

File: Podfile.lock [hunk 1]
```diff
@@ -904,7 +904,7 @@ SPEC CHECKSUMS:
   Folly: b73c3869541e86821df3c387eb0af5f65addfab4
   Forgeries: 64ced144ea8341d89a7eec9d1d7986f0f1366250
   FXBlurView: 5121730176fd52bcf7bffd0bd8c1485e8aabc3cb
-  glog: 1f3da668190260b06b429bb211bfbee5cd790c28
+  glog: 40a13f7840415b9a77023fbcae0f1e6f43192af3
```
Comment (ashleyjelks) [inline Podfile.lock:907]: Would the app not compile without this? I was under the impression that we should avoid Podfile.lock commits unless that's the case. cc @artsy/collector-experience maybe someone on this team clarify?

**Review by ashleyjelks — APPROVED** [review]

File: Podfile.lock (hunk 1 above)
Comment (xtina-starr) [reply Podfile.lock:907]: Yeah that's a good question. Should these not be checked in?

Comment (xtina-starr) [reply Podfile.lock:907]: @zephraph do you know if this is needed?

Comment (ashleyjelks) [reply Podfile.lock:907]: I saw a change to this file after making some change recently, and to test my assumption, I stashed it to find that the app would not compile without it, in which case it made sense to commit the change. I've also tried that and seen the app work as expected without the new changes to this file, so I'm not sure what the best practice here is. But do you know if the app still builds without this change?

**Pull request merged**